### PR TITLE
move couch2pg from stop -> assess

### PR DESCRIPTION
**Pull request by mrjones-plip**
Not quite sure if we're ready for this or if this is the right content, but noted it shouldn't be in `stop` anymore given it's actively being revived.

### CODE REVIEW

**Comment by andrablaj** [comment]
@mrjones-plip, thank you for starting the discussion here; this is an excellent topic!

The @medic/ecosystem decided to revamp `couch2pg` under CHT Sync, meaning that we won't add new code to the existing `couch2pg` repo, but we will adjust CHT Sync to be a new version of `couch2pg` + make use of CHT Pipeline. The reasoning is not to be too disruptive of the community members who use `couch2pg`, as the introduction of CHT Pipeline is a breaking change. 

In conclusion, I suggest we keep `couch2pg` in `stop` adoption degree, and we update the CHT Sync & CHT Pipeline blips to mention that the logic behind the sync used `couch2pg` logic.

**Comment by mrjones-plip** [comment]
@andrablaj - awesome!  that all makes sense. this PR was fun to learn how to update the radar, but I'll close since it's not the right content nor the right time ;)

**Comment by andrablaj** [comment]
I appreciate you opening this PR @mrjones-plip, as this is the right attitude, and this is why this tool exists: to question and stay up to date with tech! 💪  Don't hesitate to open other PRs or issues if you see inconsistencies in the current version.
